extensions/amp-form/0.1/amp-form.js

```diff
@@ -566,16 +566,15 @@ export class AmpForm {
   /**
    * Handle successful presubmit tasks
    * @param {!ActionTrust} trust
-   * @param {?Event} event
    * @return {!Promise}
    */
-  handlePresubmitSuccess_(trust, event) {
+  handlePresubmitSuccess_(trust) {
     if (this.xhrAction_) {
       return this.handleXhrSubmit_(trust);
     } else if (this.method_ == 'POST') {
       this.handleNonXhrPost_();
     } else if (this.method_ == 'GET') {
-      this.handleNonXhrGet_(event);
+      this.handleNonXhrGet_(true);
```

Comment:
nit: add a boolean explainer comment `this.handleNonXhrGet_(/*shouldSubmitFormElement*/ true);`